cranelift-frontend/src/frontend.rs

```diff
@@ -628,15 +627,23 @@ impl<'a> FunctionBuilder<'a> {
         self.ins().call(libc_memcpy, &[dest, src, size]);
     }
 
-    /// Optimised memcpy for small copies.
-    pub fn emit_small_memcpy(
+    /// Optimised memcpy or memmove for small copies.
+    ///
+    /// # Codegen safety
+    ///
+    /// The following properties must hold to prevent UB:
+    ///
+    /// * `src_align` and `dest_align` are an upper-bound on the alignment of `src` cq `dest`.
```

Comment:
`src cq dest`

Is `cq` a typo or am I missing an abbreviation?

Reply:
It seems that `cq` is a Dutch abbreviation: https://nl.m.wikipedia.org/wiki/Casu_quo. I also used it wrong. :man_facepalming: I meant: `src_align` is the upper-bound on the alignment of `src` and `dest_align` is the upper-bound on the alignment of `dest`. I replaced it with `respectively`.